modules/pool/pool.gql

```diff
@@ -983,11 +995,6 @@ input GqlPoolFilter {
     minTvl: Float
 }
 
-enum GqlPoolFilterCategory {
-    INCENTIVIZED
-    BLACK_LISTED
-}
-
```

Comment:
this is a breaking change

Reply:
Good catch, it was renamed by mistake. The GqlPoolFilterCategory is back.